docs/configure_admin_workstation_post_install.rst

```diff
@@ -63,3 +63,8 @@ of the authenticated hidden services, restart Tails and make
 sure to enable persistence.
 
 .. _Authenticated Tor Hidden Services: https://www.torproject.org/docs/tor-manual.html.en#HiddenServiceAuthorizeClient
+
+Back Up The Workstations
+------------------------------------
+
+USB drives can wear out, get lost, or otherwise become corrupted, making it very important to be sure to keep current backups. Follow the :ref:`Backup the Workstations <backup_workstations>` document to create a backup of your Secure Viewing Station, Admin Workstation, and Journalist Workstations after you've completed the installation and post-installation steps.
```

Comment:
Nit: We italicize SecureDrop-specific terminology (see https://docs.securedrop.org/en/release-0.12.1/development/documentation_guidelines.html and https://docs.securedrop.org/en/release-0.12.1/terminology.html ).